src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -26,17 +26,21 @@
 package com.sun.net.httpserver;
 
 import java.io.IOException;
-import java.util.*;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.ListIterator;
 
 /**
  * A filter used to pre- and post-process incoming requests. Pre-processing occurs
  * before the application's exchange handler is invoked, and post-processing
  * occurs after the exchange handler returns.  Filters
- * are organised in chains, and are associated with HttpContext instances.
- * <p>
- * Each Filter in the chain, invokes the next filter within its own
- * doFilter() implementation. The final Filter in the chain invokes the applications
- * exchange handler.
+ * are organised in chains, and are associated with {@link HttpContext} instances.
+ *
+ * <p> Each Filter in the chain, invokes the next filter within its own
```

Comment:
Here `Each Filter` should be either `Each filter` or `Each {@code Filter}`

Reply:
Change made in commit 4de7e6c